zgw/object-management/src/main/resources/config/liquibase/changelog/20221221-add-object-management-table.xml

```diff
@@ -50,4 +50,13 @@
     <changeSet id="objecttype_id unique in object_management_configuration" author="Ritense">
         <addUniqueConstraint tableName="object_management_configuration" columnNames="objecttype_id" constraintName="uc_object_management_configuration_objecttype"/>
     </changeSet>
+
+    <changeSet id="add version column to object_management_configuration" author="Ritense">
+        <addColumn tableName="object_management_configuration">
```

Comment:
I think this was okay as object management isn't intended to be used yet... but just to be nice, maybe we can include a changeset that clears this table?

Reply:
@ThomasRitense Do we need to clear the objectmanagement table, or not?

Reply:
imo, the `defaultValue="1"` is good enough. Since that makes sure nothing breaks

Reply:
Agree with Klaas. This is not used in production yet and probably not even on test/acceptance.